### PR TITLE
Damacus/patch libraries

### PR DESCRIPTION
**Pull request by damacus**
### Description

Get server_install working. 

### Issues Resolved

Server install for centos-7 now works locally.
### Contribution Check List

- [ ] All tests pass.
- [ ] New functionality includes testing.
- [ ] New functionality has been documented in the README if applicable

### CODE REVIEW

File: test/cookbooks/test/recipes/server_install.rb [hunk 1]
```diff
@@ -8,7 +8,18 @@
   version '9.5'
 end
 
+pg_config = {
+  'default_statistics_target' => '100',
+  'maintenance_work_mem'      => "512MB",
```
Comment (martinisoft) [inline test/cookbooks/test/recipes/server_install.rb]: Would switch the double quotes to single for consistency here

File: resources/server_install.rb [hunk 2]
```diff
@@ -25,6 +25,8 @@
 property :port,              [String, Integer], default: 5432
 property :initdb_locale,     String, default: 'UTF-8'
 
+include PostgresqlCookbook::Helpers
```
Comment (martinisoft) [inline resources/server_install.rb]: Were these not included before? Why are they being added now?

File: metadata.rb [hunk 3]
```diff
@@ -5,10 +5,10 @@
 license           'Apache-2.0'
 description       'Installs and configures postgresql for clients or servers'
 long_description  IO.read(File.join(File.dirname(__FILE__), 'README.md'))
-version           '6.1.2'
+version           '7.0.0'
```
Comment (martinisoft) [inline metadata.rb]: Do we need to bump the version now?

Comment (damacus) [reply metadata.rb]: Not really, but it reminds us/people that we're no longer aiming for 6.x series.

File: resources/server_install.rb (hunk 2 above)
Comment (martinisoft) [reply resources/server_install.rb]: Ah. In that case I'd add it in to `action_class`.

Comment (damacus) [reply resources/server_install.rb]: The action_class block is the provider portion of the file. So it only works for properties methods in the `action` block. Now I think of it, that's probably why it's called the action class.

Comment (martinisoft) [reply resources/server_install.rb]: Oh right yeah you need that for a property, nevermind then.

**Pull request closed without merging**